README.md

```diff
@@ -8,16 +8,18 @@ It does not use zero-installs feature - file `.pnp.cjs` and directory `.yarn/cac
 - [x] Semver
 - [ ] Tag
 - [x] Npm alias
-- [x] Git
+- [ ] ~~Git~~
 - [x] GitHub
```

Comment:
Haven't we dropped GitHub as well?

Reply:
Good catch, I forgot those are documented as separate protocols